frame/examples/offchain-worker/src/lib.rs

```diff
@@ -111,6 +111,8 @@ pub mod crypto {
 	}
 }
 
+const MAX_PRICES_LEN: u32 = 64;
```

Comment:
Why did you opt for a const here instead of putting it in the config?

Reply:
because that was the way it was before :/

Reply:
ftfy  
*git cherry-pick 5f6ff0d3cf700f19def8ded525ff72488ee364d1*

Reply:
There is still a `#[pallet::without_storage_info]` in the `offchain-worker`. I also fixed that in the commit.